File: apps/web/src/auth/role-labels.ts
/** PT-BR role labels — UI copy only; code and API stay on English enums. */
import type { AnyRoleName } from '@tatame/shared';

export const ROLE_LABELS: Record<AnyRoleName, string> = {
  student: 'Aluno',
  professor: 'Professor',
  admin: 'Admin',
  guardian: 'Responsável',
  owner: 'Owner',
  support: 'Suporte',
  finance: 'Financeiro',
};

File: apps/mobile-ios/Packages/TatameCore/Sources/TatameCore/RoleLabels.swift
// PT-BR display labels (UI copy only — code and enums stay English, per the
// charter's language rule). Shared by the login membership chooser and the
// persona shells.

public extension MembershipRole {
    /// Handoff copy for the role name.
    var displayNamePTBR: String {
        switch self {
        case .student: "Aluno"
        case .professor: "Professor"
        case .admin: "Admin da academia"
        case .guardian: "Responsável"
        case .owner: "Plataforma — Owner"
        case .support: "Plataforma — Suporte"
        case .finance: "Plataforma — Financeiro"
        }
    }
}
